src/views/accounts/Account/AccountDetail/Detail.vue

```diff
@@ -262,6 +257,14 @@ export default {
       ).then(res => {
         openTaskPage(res['task'])
       })
+    },
+    handleClick() {
+      this.drawerRefName = 'AssetDetail'
+      this.$route.params.id = this.object.asset.id
+
+      this.$nextTick(() => {
+        this.pamDrawerShow = true
+      })
     }
   }
 }
```

Comment:
The code has some inconsistencies between different parts, such as the variable names "this" vs "this object". It also seems there's an error in line 270 while defining the router components:

```javascript
export default {
    // ...
        routes:[
            {...routes},
             {
                path:'/assetdetail/:id',
                component:Component,
                 methods:{'handleClick':()=>'{}'},
                 props:{
                   object:Object
                    } 
            },{
                path:'/',
```
Make sure you have correctly assigned variables like this:

```javascript
export default {
    // ... other lines here...
}
```

This will prevent any unexpected errors from appearing in certain situations.